specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/iotDefenderSettings.json

```diff
@@ -0,0 +1,445 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2020-08-06-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/iotDefenderSettings": {
```

Comment:
>iotDefenderSettings [](start = 66, length = 19)

Can you explain the scenario? Why does it needs to be subscription level? Can this be different per resource group? How about reusing it across a Management group?

Reply:
The service ("Defender for IoT") is similar in nature to Azure Security Center while also operates in a [subscription level](https://docs.microsoft.com/en-us/azure/security-center/security-center-pricing). So some of our (proxy) resource types are also maintained at the subscription level.